packages/perseus/src/perseus-types.ts

```diff
@@ -1275,6 +1278,20 @@ export type PerseusCSProgramSetting = {
     value: string;
 };
 
+export type PerseusPythonProgramWidgetOptions = {
+    // The ID of the Python program to embed
+    programID: string;
+    // The height of the widget
+    height: number | string;
+}
+
+// export type PerseusPythonIFrameWidgetOptions = {
+//     // A Python Program ID
+//     url: string;
+//     // The height of the widget
+//     height: number | string;
+// }
+
```

Comment:
Can we remove this type also? `PerseusPythonProgramWidgetOptions` looks good to me.